packages-next/keystone/src/lib/createAdminUIServer.ts

```diff
@@ -36,14 +36,14 @@ const addApolloServer = ({ server, system }: { server: any; system: KeystoneSyst
   });
   // FIXME: Support custom API path via config.graphql.path.
   // Note: Core keystone uses '/admin/api' as the default.
-  apolloServer.applyMiddleware({ app: server, path: '/api/graphql' });
+  apolloServer.applyMiddleware({ app: server, path: '/api/graphql', cors: false });
 };
 
-export const createAdminUIServer = async (system: KeystoneSystem) => {
+export const createAdminUIServer = async (config: KeystoneConfig, system: KeystoneSystem) => {
   const server = express();
 
   // TODO: allow cors to be configured
-  server.use(cors({ origin: true, credentials: true }));
+  server.use(cors({ origin: true, credentials: true, ...(config.server?.cors || {}) }));
```

Comment:
I would not include cors by default like this, aside from it being an opinionated thing to do, you're also not able to turn it off.

Including defaults is (probably) fine, if they make sense, I'm not sure these are the right ones.

I'd basically not include this line at all if `config.server.cors` isn't defined, and _maybe_ (probably _later_) allow it to be `true` for the defaults and an object if you want to specify your own config (specifying would **set** not **spread**)